docs/developers/setting-up-metamask.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Setting up Metamask
```

Comment:
We should standardise on double capital M everywhere. Search and replace is your friend!